playbook/app/pb_kits/playbook/pb_typeahead/docs/_typeahead_with_pills_async_custom_options.jsx

```diff
@@ -0,0 +1,100 @@
+/* @flow */
+
+import React, { useState } from 'react'
+
+import {
+    Caption,
+    Typeahead,
+    User,
+} from '../..'
+
+/**
+ *
+ * @const filterResults
+ * @ignore
+ * @returns {[Object]} - a custom mapping of objects, minimally containing
+ * `value` and `label` among other possible fields
+ * @summary - for doc example purposes only
+ */
+
+const filterResults = (results) =>
+  results.items.map((result) => {
+    return {
+      imageUrl: result.avatar_url, //add the custom field
```

Comment:
Anything attribute-wise coming from Nitro should not be `avatar`, but `photo` related. Just make sure that if you're adding a new value, that the URL is coming from `photo`. Anything `avatar` related is deprecated and I'm going through this week trying to stamp out all occurrences of this.

Reply:
Good to know. In this case, this is just an example of how to use the typeahead kit, and we are using Github API to exemplify it, so I don't think this will affect Nitro. Thanks Wade